### PR TITLE
Alkalid proposal: lowbit

### PR DESCRIPTION
**Pull request by Alkalid**
---
A good PR should include the following parts:

- [ ] A clear title
- [ ] A meaningful message for PR, as well as its commits
- [ ] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [ ] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

File: project/Alkalid/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# ULTRA LOW BIT: The Multiple One-Bit Quantization
+
+## Basic Information
+Artificial Intelligence (AI) applications have become widespread, and in recent years, they have been extensively applied on edge devices. However, the computational demands of deep learning models are often too great for such devices to handle. To address this, many applications use a method called quantization, which reduces model parameters to 8 bits (integer) or even fewer, such as one bit. One of the biggest advantages of one-bit quantization is that binary multiplication can be simplified to an XNOR operation, which is more efficient in SIMD computations.
```
Comment (yungyuc) [inline project/Alkalid/README.md:4]: Please use a proper length (e.g., 79 characters) to break lines.  It's hard to review without proper line break.

File: project/Alkalid/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Problem to Solve
+Running deep learning models on edge devices with one-bit quantization greatly benefits computational efficiency but also reduces the model's precision significantly, which in turn, can lead to a substantial decrease in accuracy. Therefore, I aim to implement a "multiple one-bit" algorithm to improve precision while still maintaining the advantages of XNOR instructions.
```
Comment (yungyuc) [inline project/Alkalid/README.md:7]: What is the value range before the quantification?

**Review by yungyuc — CHANGES_REQUESTED** [review]
- [ ] Breaks lines properly

**Comment by yungyuc** [comment]
@Alkalid Please leave a global comment and resolve merge conflict when requesting the next review.

Do you have time to join the meeting this week https://sciwork.dev/meetup/2023/1108-nycu ?  Please provide me your update in the discord #nsd channel by tagging me.

- [ ] Resolve merge conflict.

**Review by Alkalid — COMMENTED** [review]
@yungyuc This is my reply. Please could you have a look.

File: project/Alkalid/README.md (hunk 1 above)
Comment (yungyuc) [reply project/Alkalid/README.md]: It looks nicer.

**Review by yungyuc — CHANGES_REQUESTED** [review]
It's an OK plan.  Once you provide the project github repository, I can merge.  But I am not sure the problem may give you a story good enough.  We'll see.

- [x] Add link to your project github repository.

File: project/Alkalid/README.md (hunk 2 above)
Comment (Alkalid) [reply project/Alkalid/README.md]: No problem, I added it

**Review by Alkalid — COMMENTED** [review]
@yungyuc Here is my reply. Please could you have a look?

**Comment by yungyuc** [comment]
LGTM.  Please start to add code.  Your repository is empty and he must feel very lonely.

**Pull request merged**